docs/source/getting-started/quickstart.rs

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env -S cargo +nightly -Zscript
+```cargo
+# init
+[dependencies]
+opendp = { version = "0.9.2", features = ["contrib", "honest-but-curious"] }
```

Comment:
I think this should be a relative path:
* We'd rather not need to update these version numbers throughout the scripts
* We'd like to be able to test things that haven't been released yet

Reply:
The downside here is that we'll need a separate init block. Think we could have one init block in the quickstart, and omit init blocks elsewhere?

Reply:
Updated: Now there's the real cargo block that uses the path, and suggested one that will be included in the docs.